backend/webserver/views.py

```diff
@@ -433,13 +508,18 @@ def get(self, request, author_id):
             )
         else:
             serializer = self.get_serializer(request, queryset)
+        for data in serializer.data:
```

Comment:
this piece of code should not reside here. unnecessarily iterating over the  already serialized data just make one change makes for slow code. it should live in the `PostLikeSerializer` by overriding the `to_representation` method. But this is fine for now, I will refactor it later.